src/control/mobile_manipulation/include/mobile_manipulation_rl/worlds.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
```

Comment:
I am not 100% sure about the C/C++ coding style (perhaps @stefaniapedrazzi can answer this), but I think that we should have a uniform way of structuring header files. Some files uses traditional include guards, while this one uses a *pragma* directive.

Reply:
Happy to adjust if someone can confirm which convention to use.

Reply:
I do not have a preference. Theoretically, the traditional include guards are supported by every compiler, so they are more portable (so I would tend towards this for this reason). However, I also think that every major compiler also supports pragma directives so this is probably not a real concern.

Reply:
Done, using traditional guards consistently now